x-pack/plugins/ml/server/lib/register_settings.ts

```diff
@@ -30,5 +34,41 @@ export function registerKibanaSettings(coreSetup: CoreSetup) {
         }),
       },
     },
+    [ANOMALY_DETECTION_ENABLE_TIME_RANGE]: {
+      name: i18n.translate('xpack.ml.advancedSettings.enableAnomalyDetectionDefaultTimeRangeName', {
+        defaultMessage: 'Enable default time range for anomaly detection jobs.',
+      }),
+      value: false,
+      schema: schema.boolean(),
+      description: i18n.translate(
+        'xpack.ml.advancedSettings.anomalyDetectionDefaultTimeRangeDesc',
+        {
+          defaultMessage: 'Use a default time range to view anomaly detection jobs.',
```

Comment:
If I'm understanding correctly that this applies only to specific interfaces, I suggest we mention them here:
```suggestion
          defaultMessage: 'Use the default time filter in the Single Metric Viewer and Anomaly Explorer.',
```

Reply:
Updated here https://github.com/elastic/kibana/pull/76347/commits/41af491d1fa355395c34966f6c17b1275917b153